### PR TITLE
Move turbo cartocss

### PR DESCRIPTION
**Pull request by dgaubert**
cc @rochoa

### CODE REVIEW

File: lib/cartodb/controllers/map.js [hunk 1]
```diff
@@ -162,6 +163,23 @@ MapController.prototype.create = function(req, res, prepareConfigFn) {
                 if (layers) {
                     requestMapConfig.layers = layers;
                 }
+
+                return next(null, requestMapConfig, datasource);
+            });
+        },
+        function parseTurboCartoCss(err, requestMapConfig, datasource) {
```
Comment (rochoa) [inline lib/cartodb/controllers/map.js]: We need to apply this to named maps also, right?

**Comment by rochoa** [comment]
I'm missing the turbo-cartocss replacement for named maps, apart from that ... LGTM! :+1:

File: lib/cartodb/controllers/map.js (hunk 1 above)
Comment (dgaubert) [inline lib/cartodb/controllers/map.js]: Done! 

I don't know why, but I missed it. :sweat_smile:

**Comment by dgaubert** [comment]
Finally, integration of turbo-cartocss for named maps is done. It took more time than expected, I had to dedicate several hours to understand how templates, named-layers and named-maps work behind the scenes.

:tired_face:

File: lib/cartodb/models/mapconfig/named_map_provider.js [hunk 2]
```diff
@@ -91,6 +92,18 @@ NamedMapMapConfigProvider.prototype.getMapConfig = function(callback) {
                 }
             );
         },
+        function parseTurboCartoCss(err, _mapConfig, datasource) {
```
Comment (rochoa) [inline lib/cartodb/models/mapconfig/named_map_provider.js]: Is this what really implements turbo-cartocss in named maps, right?

File: lib/cartodb/controllers/named_maps_admin.js [hunk 3]
```diff
@@ -65,14 +65,15 @@ NamedMapsAdminController.prototype.update = function(req, res) {
...
+
     step(
         function checkPerms(){
             self.authApi.authorizedByAPIKey(cdbuser, req, this);
         },
         function updateTemplate(err, authenticated) {
             assert.ifError(err);
-            ifUnauthenticated(authenticated, 'Only authenticated user can update templated maps');
-            ifInvalidContentType(req, 'template PUT data must be of type application/json');
+            ifUnauthenticated(authenticated, 'Only authenticated users can get template maps');
```
Comment (rochoa) [inline lib/cartodb/controllers/named_maps_admin.js]: ???

Comment (dgaubert) [inline lib/cartodb/controllers/named_maps_admin.js]: :scream:

File: lib/cartodb/models/mapconfig/named_map_provider.js (hunk 2 above)
Comment (dgaubert) [inline lib/cartodb/models/mapconfig/named_map_provider.js]: Yes, it is.

**Comment by rochoa** [comment]
:shipit:

**Pull request merged**